src/04-arrays.js

```diff
@@ -92,7 +105,10 @@ function addElementToEndOfArray(array, element) {}
  *  removeElementFromEndOfArray([ 10, 9, 8 ]);
  *  //> 8
  */
-function removeElementFromEndOfArray(array) {}
+function removeElementFromEndOfArray(array) {
+  array=array.pop();
```

Comment:
This technically works but there are some confusing aspects we can smooth out.  First ` .pop() ` already returns whatever item is removed from the array on which it is called.  We could write ` return array.pop() `  but if we do want to save our value in a variable we should choose a different name for our popped off index than ` array `.  Something like ` removed ` or ` index ` is better.